portal/portal-sprint-107.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: /release-notes/portal/sprint-107
+title: Sprint 107 (August 23, 2021)
+---
+
+# fiskaltrust.Portal - Sprint 107
+_August 23, 2021_
+
+This Sprint's main goal was to enforce access rights on working with Agencies, as well as to improve the DATEV MeinFiskal onboarding process.
+
+## Features
+
+### Middleware Configuration
+
+- [Reworked Quote and Invoices Pages](#reworked-quote-and-invoices-pages)
+- [Securing Agency Page](#securing-agency-page)
+- [Archive Auditable for Outlets FR](#archive-auditable-for-outlets-fr)
+
+### Knowledge-based Articles
+
+- [Knowledge articles page improvements](#knowledge-articles-page-improvements)
+
+### Various Improvements
+
+- [DATEV MeinFiskal Improvements](#datev-meinfiskal-improvements)
+- [Contracting emails improvements](#contracting-emails-improvements)
+
+## Middleware Configuration
+
+### Reworked Quote and Invoices Pages
+
+Quotes Page has been reworked and improved. The layout has been modified in a way that the information is better aligned. The latest quotes, invoices and orders are displayed in a clearer way and are sorted descending by date (default sorting), improving the user experience for all Portal users. 
+
+![invoices](images/sprint-107/invoices.png)
+
+![quotes](images/sprint-107/quotes.png)
+
+### Securing Agency Page
+
+The Agency page has been reworked and is now properly secured. The user that is trying to load or update agencies, must have respectively the necessary access/read and access/write rights. In case those rights have not been granted, the create/edit button is now hidden. the user hasn't got the necessary access rights, the create/edit button should be hidden. 
```

Comment:
The sentence is pretty weird, I'd remove the second part 
```suggestion
The Agency page has been reworked and is now properly secured. The user that is trying to load or update agencies, must have respectively the necessary access/read and access/write rights. In case those rights have not been granted, the create/edit button is now hidden. 
```